### PR TITLE
Fix queries when Noosphere is off

### LINKED ISSUE #580: Notes screen shows spinner for fresh app install when Noosphere disabled
Reported by @jsantell.

> during the FRE on a fresh install (noosphere disabled) the first screen was the notes view with a loading widget spinning -- it wasn't until i went to settings, turned on noosphere, and went back, did the FRE (pretty new splashscreen!) start

### PR DESCRIPTION
**Pull request by gordonbrander**
Fixes #580

Regression from #561.

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/AppView.swift [hunk 1]
```diff
@@ -731,9 +731,6 @@ struct AppModel: ModelProtocol {
         logger.debug(
             "Sphere ID: \(sphereIdentity)"
         )
-        logger.debug(
```
Comment (gordonbrander) [inline xcode/Subconscious/Shared/Components/AppView.swift:734]: Discovered this redundant log line while troubleshooting.

**Review by cdata — APPROVED** [review]
LGTM

File: xcode/Subconscious/SubconsciousTests/Tests_DataService.swift [hunk 2]
```diff
@@ -219,7 +219,7 @@ final class Tests_DataService: XCTestCase {
             noosphere: noosphere,
             database: database
         )
-
+
```
[the remark this answers is not in the document]
Comment (gordonbrander) [reply xcode/Subconscious/SubconsciousTests/Tests_DataService.swift:222]: I know. So frustrating. It's impossible to fight it as well.

File: xcode/Subconscious/Shared/Models/Slashlink.swift [hunk 3]
```diff
@@ -209,8 +209,14 @@ extension Slashlink {
...
     /// Otherwise, returns slashlink unchanged.
+    ///
+    /// As a convenience, `did` may be nil. This is for cases where user does
+    /// not have a sphere. If `did` is nil, no relativization will occur.
+    ///
+    /// - Parameters:
+    ///   - did: the base to relativize to. May be nil for "no sphere".
     /// - Returns Slashlink
-    func relativizeIfNeeded(did base: Did) -> Slashlink {
+    func relativizeIfNeeded(did base: Did?) -> Slashlink {
```
[the remark this answers is not in the document]
Comment (gordonbrander) [reply xcode/Subconscious/Shared/Models/Slashlink.swift:219]: Added additional test to cover nil case

File: xcode/Subconscious/SubconsciousTests/Tests_Slashlink.swift [hunk 4]
```diff
@@ -233,7 +233,26 @@ final class Tests_Slashlink: XCTestCase {
             "Does not relativize when did is other sphere"
         )
     }
-
+    
+    func testRelativizeIfNeededNilDid() throws {
+        let did = try Did("did:key:z6MkmCJAZansQ3p1Qwx6wrF4c64yt2rcM8wMrH5Rh7DGb2K7")
```
Comment (cdata) [inline xcode/Subconscious/SubconsciousTests/Tests_Slashlink.swift:238]: Is this first `did` variable used in the test?

**Review by cdata — APPROVED** [review]

File: xcode/Subconscious/SubconsciousTests/Tests_Slashlink.swift (hunk 4 above)
Comment (gordonbrander) [reply xcode/Subconscious/SubconsciousTests/Tests_Slashlink.swift:238]: Fixed

**Pull request merged**